index.js

```diff
@@ -183,29 +183,29 @@ function update(tags, filebuffer, options, fn) {
 
     const updateFn = (currentTags) => {
         currentTags = currentTags.raw || {}
-        Object.keys(rawTags).map((specName) => {
-            const options = ID3Util.getSpecOptions(specName, 3)
+        Object.keys(rawTags).map((frameIdentifier) => {

```

Comment:
It is usually a good idea to separate the rename in a different commit, it makes it easier to review as we can skip the rename commit and focus on the other changes.